src/Account.ts

```diff
@@ -43,9 +43,16 @@ export default class Account {
    * @param password The password required to unlock the account.
    * @returns `true` if its unlocked otherwise false.
    */
-  // public unlock(web3: Web3, password: string): boolean {
-  //   // Unlocking the account and adding it to the local web3 instance so that everything is signed
-  //   // locally when using web3.eth.send
-  //   return false;
-  // }
+  public unlock(web3: Web3, password: string): boolean {
```

Comment:
Why this function is not integrated / called from anywhere else?

Reply:
The calls to this are in a separate PR.